### ISSUE TITLE
Content Request: Running Azure Functions in Containers

### ISSUE THREAD

**Issue by lechnerc77** [marcduiker/azure-functions-university#157]
### 📝 Lesson Description

This lesson should cover the following topics:

- How to build and run Azure Functions in Containers
- Constraints concerning bindings
- How to deploy the Functions to a _local_ Kubernetes cluster
- Outlook/references to further useful topics in this area (KEDA, dapr Bindings)

### ✍ Authoring of the lesson - Writing

Yes I can

### 👩‍🏫 Authoring of the lesson - Presenting

Yes

**Comment by marcduiker** [comment]
Yes @lechnerc77 , great idea! It could definitely be a series of lessons, so start small 👍
